python/sglang/srt/server_args.py

```diff
@@ -1097,6 +1103,12 @@ def add_cli_args(parser: argparse.ArgumentParser):
             action="store_true",
             help="Enabling DeepEP MoE implementation for EP MoE.",
         )
+        parser.add_argument(
+            "--moe-dense-tp-size",
+            type=int,
+            default=ServerArgs.moe_dense_tp_size,
+            help="tp_size for MoE dense MLP layers",
```

Comment:
Could you provide a more detailed explanation, stating that it is to address the issue where, with a large TP_SIZE, the dimensions of the tensors for each rank after splitting DeepseekV2MLP cannot be divided evenly by the FP8 MoE blocksize? Should the name include "Deepseek," or should we consider all MoE models?